flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/nodes/exec/stream/IntervalJoinJsonPlanTest.java

```diff
@@ -82,18 +82,19 @@ public void testProcessingTimeInnerJoinWithOnClause() {
 
     @Test
     public void testRowTimeInnerJoinWithOnClause() {
-        String sinkTableDdl =
-                "CREATE TABLE MySink (\n"
-                        + "  a int,\n"
-                        + "  b varchar\n"
-                        + ") with (\n"
-                        + "  'connector' = 'values',\n"
-                        + "  'table-sink-class' = 'DEFAULT')";
-        tEnv.executeSql(sinkTableDdl);
         util.verifyJsonPlan(
                 "INSERT INTO MySink "
                         + "SELECT t1.a, t2.b FROM A t1 JOIN B t2 ON\n"
                         + "  t1.a = t2.a AND\n"
                         + "  t1.rowtime BETWEEN t2.rowtime - INTERVAL '10' SECOND AND t2.rowtime + INTERVAL '1' HOUR");
     }
+
+    @Test
+    public void testRowTimeEquiJoin() {
```

Comment:
We can drop this test. A regular plan test via xml should be sufficient compared to a full JSON plan. If not already present, you can add this to the test that verifies `IntervalJoinTest.xml`

Reply:
We have a test for that in the xml tests https://github.com/apache/flink/pull/17369/files#diff-32daba6c5665d7991e66e5f5b3754c519e565557e7dafb11109a1cfea8e72dffR199. The json test is removed